### DISCUSSION TITLE
Disable mobs spawning with armor setting

### DISCUSSION THREAD

**Discussion by ClxdyWindows** [timbru31/SilkSpawners#515] [Ideas]
Would be awesome if this would added to config. Just a setting which would disable mobs spawned from spawners having armor. Can help with entity lag on some servers.

**Comment by timbru31** [comment]
This could probably with achieved with the `/data` command. SilkSpawners has not and will not alter spawning behavior. Probably define a static `ArmorItems` NBT tag.
Refs:
- https://minecraft.wiki/w/Monster_Spawner
- https://minecraft.wiki/w/Tutorials/Command_blocks_and_functions#Part_2:_Custom_monsters
